code/game/objects/items/AI_modules.dm

```diff
@@ -362,7 +362,7 @@ AI MODULES
 /obj/item/aiModule/core/full/asimov
 	name = "'Asimov' Core AI Module"
 	law_id = "asimov"
-	var/subject = "human being"
+	var/subject = "person of an NT approved crew species"		//CITADEL CHANGED FROM HUMANS!

```

Comment:
Pretty verbose, this equivales to `NT approved crew species` (or `specimen`) and still as vague either way without explicit reference to the concept of roundstart races.
Also you may want to replace `[subject]s` with `[subject]\s` at line 373.